packages/code-du-travail-frontend/server/koaServer.js

```diff
@@ -6,8 +6,8 @@ const Sentry = require("@sentry/node");
 const redirects = require("./redirects.json");
 const { logger } = require("@socialgouv/cdtn-logger");
 
-const IS_PRODUCTION_DEPLOYMENT =
-  process.env.IS_PRODUCTION_DEPLOYMENT === "true";
+const NEXT_PUBLIC_IS_PRODUCTION_DEPLOYMENT =
```

Comment:
Idem ici, on peut garder l'ancien naming

```suggestion
const IS_PRODUCTION_DEPLOYMENT =
```